pkg/virtctl/vm/add_volume_test.go

```diff
@@ -57,41 +57,16 @@ var _ = Describe("Add volume command", func() {
 		coreClient = fake.NewSimpleClientset()
 	})
 
-	createTestDataVolume := func() *v1beta1.DataVolume {
```

Comment:
We can possibly replace this by libdv altogether.

Reply:
it will generate a random name, and we need a known name
unless libdv will be extended by adding a name option, it can be done on a follow up of course
but this PR purpose is just to unline functions that arent needed to be lambda

Reply:
added it, despite i think it is not this PR scope